### ISSUE TITLE
Support of GeForce Now

### ISSUE THREAD

**Issue by sradola** [dbaumgarten/yodk#76]
The feature with Visual Code to paste the code directly into the Starbase Yolol chip is not working with GeForce Now.
When I remember correctly, when I want to paste the code, your extension look for an active window with the name "Starbase".
This will not work, when people play Startbase over GeForce Now. When I use an Apple Computer, are the Shortcuts still the same as by an windows pc?

**Comment by dbaumgarten** [comment]
Exactly. It does it this way to not pollute any other application with the shortcuts.
I have no idea if this would even work with GeForceNow (even when ignoring the window-title).

I guess the only way to find out would be to try it. But I don't have GeForceNow (and don't intend to get it), so I can't test it.

The shortcuts simply won't work on Mac. This whole stuff directly uses the win32 API (which clearly won't work on OSX).
Something similar would probably possible with osx, but without a Mac I am unable to test this (and this will require a lot of testing).

If someone with access to a Mac and/or GeForce now is willing to add and test the necessary features, I would happily accept a PR. But there isn't much I could do myself.

**Comment by lizelive** [comment]
Supporting this on Linux would also be very nice. Can use xdotool for linux. Or if you want cross platform enigo is a good solution.

**Comment by dbaumgarten** [comment]
There is issue #79 for the linux-support.
But given the small amount of linux-gamers and the large list of open tickets, linux-support for the auto-typing isn't on the top of my todo-list. But PRs are always welcome ;)
